pkgs/tools/typesetting/tex/pgfplots/default.nix

```diff
@@ -19,7 +19,8 @@ stdenv.mkDerivation {
     cp -prd * $out/share/texmf-nix
   ";
 
-  meta = {
-    platforms = stdenv.lib.platforms.unix;
+  meta = with stdenv.lib; {
+    platforms = platforms.unix;
+    license = licenses.gpl2;
```

Comment:
I'd say `gpl3` for this one
https://sourceforge.net/p/pgfplots/code/ci/master/tree/doc/generic/pgfplots/README